applications/GeoMechanicsApplication/custom_utilities/ublas_utilities.cpp

```diff
@@ -0,0 +1,25 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "ublas_utilities.h"
+#include <algorithm>
+
+namespace Kratos
+{
+
+Vector UblasUtilities::CreateVector(const std::initializer_list<double>& rInitializerList)
+{
+    Vector result = ZeroVector(rInitializerList.size());
+    std::ranges::copy(rInitializerList, result.begin());
```

Comment:
This creates a vector of the correct size, fills it with zeroes, and then overwrites the elements of the vector with the values provided in `rInitializerList`. To me, the part where we fill the vector with zeroes seems redundant. Perhaps the better way to go is to set the needed capacity in advance (no need to fill anything) and then copy the provided values:
```suggestion
    auto result = Vector;
    result.reserve(rInitializerList.size());
    std::ranges::copy(rInitializerList, std::back_inserter(result));
```

Reply:
As discussed, unfortunately the ublas vector doesn't have `reserve`, so keeping it as is